Util/ClassUtils.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Acl\Util;
+
+use Doctrine\Common\Util\ClassUtils as DoctrineClassUtils;
+
+/**
+ * Class related functionality for objects that
+ * might or might not be proxy objects at the moment.
+ *
+ * @see DoctrineClassUtils
+ *
+ * @author Benjamin Eberlei <kontakt@beberlei.de>
```

Comment:
IMO you should make yourself the author since the original code is not part of this class anymore and the doctrine class has this authorship.